dashboard/test/ui/features/star_labs/step_mode.feature

```diff
@@ -2,31 +2,31 @@
 Feature: Step Mode
 
 Scenario: Step Only - Failure
-  Given I am on "http://studio.code.org/s/step/lessons/1/levels/1"
+  Given I am on "http://studio.code.org/s/step/lessons/1/levels/1?blocklyVersion=google"
     And I wait for the page to fully load
   Then element "#runButton" is hidden
     And element "#resetButton" is hidden
     And element "#stepButton" is visible
     And element "#stepButton" is not disabled
 
-  When I drag block "1" to block "5"
+  When I drag block "moveForward" to block "startBlock"
```

Comment:
The `"moveForward" id was already added to this block in a previous PR as the block definition is shared across levels.